signup.html

```diff
@@ -0,0 +1,50 @@
+<html>
+
+<head>
+    <link rel="stylesheet" href="signup.css" />
+    <link rel="stylesheet" href="style.css" />
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+</head>
+
+<body>
+    <div id="MyModal" tabindex="-1" role="dialog">
+        <div class="modal-dialog" role="document">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h5 class="modal-title"><i class="fa fa-wpforms fa-2x" aria-hidden="true"></i>CREATE NEW ACCOUNT</h5>
+                </div>
+                <div class="modal-body">
+                    <form>
+                        <div class="form-group">
+                            Name <span class="required-asterisk">*</span>
+                            <input type="name" placeholder=" " class="form-control rounded-0" id="userName" required>

```

Comment:
Suggestion: use `label` tag with input or provide `aria-label` attribute in `input` tag for accessibility